Data/ByteString/Builder/Prim.hs

```diff
@@ -433,8 +433,8 @@ module Data.ByteString.Builder.Prim (
   -- a decimal number with UTF-8 encoded characters.
   , charUtf8
 
-  , cstring
-  , cstringUtf8
+  , cstring     -- Backwards-compatibility re-exports from Internal.hs
+  , cstringUtf8 -- these no longer make use of the BoundPrim API.
```

Comment:
We still want `@since 0.11.0.0` on these in this module; make trivial wrappers here for that purpose. (We can also consider deprecating those wrappers...)

Reply:
Done. Let's squash before merging once the reviews are in.